.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "gir"]
 	path = gir
-	url = https://github.com/gtk-rs/gir
+	url = https://github.com/fengalin/gir
```

Comment:
We'll need to be careful not to merge this change. ;)

Reply:
Hehe :)